frontend/src/components/resourceMap/KubeObjectGlance/KubeObjectGlance.tsx

```diff
@@ -35,44 +37,52 @@ import { ReplicaSetGlance } from './ReplicaSetGlance';
 import { ServiceGlance } from './ServiceGlance';
 
 /**
- * Little Popup preview of a Kube object
+ * Little Popup preview of a node (Kubernetes object or other)
  */
-export const KubeObjectGlance = memo(({ resource }: { resource: KubeObject }) => {
+export const KubeObjectGlance = memo(({ resource }: { resource: any }) => {
```

Comment:
KubeObjectGlance will only render when it is a KubeObject, please check out `KubeObjectNode` component where it's used. I would suggest introducing a generic `NodeGlance` that will either render a KubeObjectGlance or a plugin provided glance from the redux state. And adding it to the KubeObjectNode component